podjs/src/tests/browserPod.test.ts

```diff
@@ -0,0 +1,6 @@
+import { podSpec } from "@polypoly-eu/pod-api";
```

Comment:
We might want to salvage this from here. `DefaultPod` can still be kept. Do you want me to do that?